src/chain.h

```diff
@@ -37,6 +38,8 @@ static constexpr int64_t TIMESTAMP_WINDOW = MAX_FUTURE_BLOCK_TIME;
  */
 static constexpr int64_t MAX_BLOCK_TIME_GAP = 90 * 60;
 
+extern RecursiveMutex cs_main;
```

Comment:
style nit, here and after: For new code maybe be explicit about the global namespace:
```suggestion
extern RecursiveMutex ::cs_main;
```
?

Reply:
Done (I think).